traits/constants.py

```diff
@@ -215,4 +219,5 @@ class DefaultValue(IntEnum):
     DefaultValue.callable_and_args: "factory",
     DefaultValue.callable: "method",
     DefaultValue.trait_set_object: "set",
+    DefaultValue.unsupported: "unsupported"
```

Comment:
The `default_kind` property is essentially unused; I'm including this only for completeness.